packages/hw-app-btc/src/Btc.js

```diff
@@ -166,6 +181,32 @@ export default class Btc {
       .then(processOutputs);
   }
 
+  async getTrustedInputBIP143(indexLookup: number, transaction: Transaction) {
+    if (!transaction) {
+      throw new Error("getTrustedInputBIP143: missing tx");
+    }
+    let sha = createHash("sha256");
+    sha.update(this.serializeTransaction(transaction, true));
+    let hash = sha.digest();
+    sha = createHash("sha256");
+    sha.update(hash);
+    hash = sha.digest();
+    const look1 = indexLookup & 0xff;
+    const look2 = (indexLookup >> 8) & 0xff;
+    const look3 = (indexLookup >> 16) & 0xff;
+    const look4 = (indexLookup >> 24) & 0xff;
+    let data = Buffer.from([look1, look2, look3, look4]);
```

Comment:
not very important but I would suggest to use `data = Buffer.alloc(4);` and then do `data.writeUInt32LE(indexLookup, 0)` . it makes it a bit less error prone and simple. 

( ultimately in the future, i wish we can find an interesting abstraction so we don't directly deal with buffer but with a descriptive specification where we express the binary format )